core/core-impl/src/test/java/io/fabric8/launcher/core/impl/filters/TokenFixtures.java

```diff
@@ -0,0 +1,53 @@
+package io.fabric8.launcher.core.impl.filters;
+
+public class TokenFixtures {
```

Comment:
If this class is meant to be used in the tests only, move it to launcher-base-test

Reply:
Fixed with 1b3cf1de